requirements.txt

```diff
@@ -1,4 +1,5 @@
 buildbot[bundle]~=2.9.1
 buildbot-worker~=2.9.1
 txrequests==0.9.6
-Twisted~=20.3.0
\ No newline at end of file
+Twisted~=20.3.0
+pywin32==300; sys_platform == 'win32'
```

Comment:
Does the sys_platform== thingy mean that this requirement gets ignored on other platforms?

Reply:
Yes, and that's intentional since pywin32 makes no sense to install on macOS or Linux.